apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -1,27 +1,34 @@
-import React, {useCallback, useEffect, useState, useRef} from 'react';
-import PropTypes from 'prop-types';
-import style from './rubrics.module.scss';
-import i18n from '@cdo/locale';
+/* eslint-disable import/order */
+// Disabling import/order rule for this file because product tour imports are grouped.
 import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, {useCallback, useEffect, useState, useRef} from 'react';
+import Draggable from 'react-draggable';
+
 import {Heading6} from '@cdo/apps/componentLibrary/typography';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import HttpClient from '@cdo/apps/util/HttpClient';
+import {tryGetSessionStorage, trySetSessionStorage} from '@cdo/apps/utils';
+import i18n from '@cdo/locale';
+import aiBotOutlineIcon from '@cdo/static/ai-bot-outline.png';
+
+import RubricContent from './RubricContent';
+import {TAB_NAMES} from './rubricHelpers';
+import RubricSettings from './RubricSettings';
 import {
   reportingDataShape,
   rubricShape,
   studentLevelInfoShape,
 } from './rubricShapes';
-import RubricContent from './RubricContent';
-import RubricSettings from './RubricSettings';
-import RubricTabButtons from './RubricTabButtons';
 import RubricSubmitFooter from './RubricSubmitFooter';
-import {tryGetSessionStorage, trySetSessionStorage} from '@cdo/apps/utils';
-import Draggable from 'react-draggable';
-import {TAB_NAMES} from './rubricHelpers';
-import aiBotOutlineIcon from '@cdo/static/ai-bot-outline.png';
-import HttpClient from '@cdo/apps/util/HttpClient';
-import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
-import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import RubricTabButtons from './RubricTabButtons';
+
+import style from './rubrics.module.scss';
 
+/* eslint-disable import/order */
```

Comment:
please also reenable the rule after this group of import! I'd rather not risk new imports getting added below this not being linted